src/main/java/frc/robot/constants/LiftConstants.java

```diff
@@ -0,0 +1,6 @@
+public class LiftConstants {
+    public static final int leftLiftMotorPort = 0;
+    public static final int rightLiftMotorPort = 1;
```

Comment:
```suggestion
    public static final int leftMotorPort = 0;
    public static final int rightMotorPort = 1;
```
Since this will be accessed as LiftConstants.leftMotorPort we don't need to name lift here